root/etc/cont-init.d/90-deprecated

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/with-contenv bash
+
+echo '
+*************************************************************
+*************************************************************
+*                                                           *
+*                                                           *
+*             This image has been deprecated                *
+*                                                           *
+*                 Use crazy-max's image at                  *
+*                                                           *
+*           crazy-max/docker-rtorrent-rutorrent             *
```

Comment:
Is this right? I think the docker part is extra

Reply:
Ah, right. That's the repo name.